nucleus/src/main/java/za/co/absa/subatomic/domain/application/Application.java

```diff
@@ -33,6 +34,12 @@
 
     @CommandHandler
     public Application(NewApplication command) {
+        if (!command.getAllAssociateProjectOwnerAndMemberIds()
+                .contains(command.getRequestedBy().getTeamMemberId())) {
+            throw new SecurityException(
```

Comment:
Assuming you're using [`java.lang.SecurityException`](https://docs.oracle.com/javase/8/docs/api/java/lang/SecurityException.html).

I'd stay clear of pigging backing off Exceptions meant for internal use. Instead, create a meaningful Gluon exception like `ApplicationAuthorisationException` that extends `RuntimeException`. You can also use Spring MVC's [`@ExceptionHandler`](http://www.baeldung.com/exception-handling-for-rest-with-spring#controlleradvice) to handle the exception and assign the correct HTTP response codes in the REST adapter.

Reply:
I even spoke to you about doing this. Then I just flat out forgot to... :) oops